llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -2940,6 +2940,14 @@ InstructionCost AArch64TTIImpl::getArithmeticInstrCost(
 
     return BaseT::getArithmeticInstrCost(Opcode, Ty, CostKind, Op1Info,
                                          Op2Info);
+  case ISD::FREM:
+    if (!Ty->isVectorTy()) {
+      Function *F =
+          CxtI == nullptr ? nullptr : CxtI->getModule()->getFunction("fmod");
```

Comment:
I am not sure that this will ever return you the right function pointer, and from what I am seeing AArch64 backend will generate frem as fmod/fmodf even if these functions aren't declared in the module, for example looking at llvm-project/llvm/test/CodeGen/AArch64/frem.ll test